### DISCUSSION TITLE
插件设置界面的sever填什么？

### DISCUSSION THREAD

**Discussion by lxlalalala** [metatube-community/jellyfin-plugin-metatube#46] [Q&A] [answered]
大佬，我用群晖的docker里的印象“javtube/javtube-server:latest”部署了后端，除了环境里的port从8080改成了8088，其他都是默认配置，插件设置界面里的sever我填的是“http://192.168.50.113:8088”，其他没动。但是在jellyfin里刷新元数据后就提示了队列一刷新，然而啥都没刮削到。是我的sever设置错误了吗？还是说nas要科学上网才能用？

**Comment by Curtain-c** [comment, marked as answer]
> 大佬，我用群晖的docker里的印象“javtube/javtube-server:latest”部署了后端，除了环境里的port从8080改成了8088，其他都是默认配置，插件设置界面里的sever我填的是“http://192.168.50.113:8088”，其他没动。但是在jellyfin里刷新元数据后就提示了队列一刷新，然而啥都没刮削到。是我的sever设置错误了吗？还是说nas要科学上网才能用？

当然得科学上网、元数据来源网站都在国外

**Reply by lxlalalala** [reply]
我这个sever地址填的没问题是吧？
那后期会不会增加个功能，可以在配置里面添加自己的ssr链接来科学上网刮削？
